simple_network_sim/sampleUseOfModel.py

```diff
@@ -23,26 +24,44 @@ def main(argv):
     setup_logger(args)
     logger.info("Parameters\n%s", "\n".join(f"\t{key}={value}" for key, value in args._get_kwargs()))
 
-    with data.Datastore(args.data_pipeline_config) as store:
+    issues: List[standard_api.Issue] = []
+
+    info = common.get_repo_info()
+    if not info.git_sha:
+        desc = "Not running from a git repo, so no git_sha associated with the run"
+        logger.warning(desc)
+        issues.append(standard_api.Issue(severity=10, description=desc))
+    elif info.is_dirty:
+        desc = "Running out of a dirty git repo"
+        logger.warning(desc)
+        issues.append(standard_api.Issue(severity=10, description=desc))
+    with standard_api.StandardAPI(args.data_pipeline_config, uri=info.uri, git_sha=info.git_sha) as store:
         network = ss.createNetworkOfPopulation(
-            store.read_table("human/compartment-transition"),
-            store.read_table("human/population"),
-            store.read_table("human/commutes"),
-            store.read_table("human/mixing-matrix"),
-            store.read_table("human/infectious-compartments"),
-            store.read_table("human/infection-probability"),
-            store.read_table("human/initial-infections"),
-            store.read_table("human/trials"),
-            store.read_table("human/movement-multipliers") if args.use_movement_multipliers else None,
-            store.read_table("human/stochastic-mode"),
-            store.read_table("human/random-seed"),
+            store.read_table("human/compartment-transition", "compartment-transition"),
+            store.read_table("human/population", "population"),
+            store.read_table("human/commutes", "commutes"),
+            store.read_table("human/mixing-matrix", "mixing-matrix"),
+            store.read_table("human/infectious-compartments", "infectious-compartments"),
+            store.read_table("human/infection-probability", "infection-probability"),
+            store.read_table("human/initial-infections", "initial-infections"),
+            store.read_table("human/trials", "trials"),
+            store.read_table("human/movement-multipliers", "movement-multipliers") if args.use_movement_multipliers else None,
+            store.read_table("human/stochastic-mode", "stochastic-mode"),
+            store.read_table("human/random-seed", "random-seed"),
         )
 
         results = runSimulation(network, args.time)
         aggregated = aggregateResults(results)
 
         logger.info("Writing output")
-        store.write_table("output/simple_network_sim/outbreak-timeseries", aggregated)
+        # The data pipeline API doesn't support categorical data, so we need to convert these to strings
+        aggregated = aggregated.astype({"node": "str", "age": "str", "state": "str"})
+        store.write_table(
+            "output/simple_network_sim/outbreak-timeseries",
+            "outbreak-timeseries",
+            aggregated,
+            issues=issues,
```

Comment:
You could add a suitable [description](https://github.com/ScottishCovidResponse/data_pipeline_api/blob/master/data_pipeline_api/standard_api.py#L247) in here as well (which I would expect to be a string constant).